src/redux/slices/productsCartSlice.js

```diff
@@ -0,0 +1,59 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+//Slice
+export const productsCartSlice = createSlice({
+  name: 'productsCartList',
+  initialState: {
+    productsCart: [],
+    cartQuantity: 0,
+  },
+  reducers: {
+    addProduct: (state, action) => {
+      for (let i = 0; i < state.productsCart.length; i++) {
```

Comment:
creo que esta logica podrias ser mas funcional usando .map o reducer

Reply:
Creo que en este caso no serviria el .map o reducer porque no estoy modificando cada uno sino solo encontrando uno de la lista, modificandolo y me salgo del for.